extension/src/experiments/model/modify/quickPick.test.ts

```diff
@@ -78,8 +109,57 @@ describe('pickAndModifyParams', () => {
       '-S',
       `params.yaml:code_names=${thirdInput}`,
       '-S',
+      `params.yaml:transforms='${fourthInput}'`,
+      '-S',
       [unchanged.path, unchanged.value].join('=')
     ])
-    expect(mockedGetInput).toHaveBeenCalledTimes(3)
+  })
+
+  it('should convert any unselected params into the required format', async () => {
```

Comment:
[F] This is actually getting dropped in the next PR as we are disabling modify and on anything that isn't the workspace (see #4088 for more details).